themes/gatsby-theme-blog-core/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gatsby-theme-blog-core",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "christopherbiscardi <chris@christopherbiscardi.com> (@chrisbiscardi)",
+  "license": "MIT",
+  "peerDependencies": {
+    "gatsby": "^2.13.13",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "dependencies": {
+    "gatsby-plugin-mdx": "^1.0.16",
+    "gatsby-plugin-sharp": "^2.2.9",
+    "gatsby-remark-copy-linked-files": "^2.1.3",
+    "gatsby-remark-images": "^3.1.7",
+    "gatsby-remark-numbered-footnotes": "^1.0.0",
+    "gatsby-remark-smartypants": "^2.1.2",
+    "gatsby-source-filesystem": "^2.1.6",
+    "gatsby-transformer-sharp": "^2.2.4",
```

Comment:
I see the benefit in having the sharp image stuff baked into the core theme, but also wonder if we should strive to make it even less opinionated by not having sharp as a dependency – and maybe fewer of the Remark plugins. In the future, I suspect we'll be able to handle some of this with components, which seems easier to change from the end-user's perspective, but we don't have those yet, so I'm fine keeping this in. Just wanted to pose the question

Reply:
Since sharp is the 3rd, 4th, and 5th most downloaded plugins I'm going to keep sharp in for now. I think that moving it somewhere else is something we should come back to in the future.

![image](https://user-images.githubusercontent.com/551247/62902108-68b17780-bd13-11e9-86c2-99c52194cdaa.png)